src/calculateRentalCost.js

```diff
@@ -29,6 +29,21 @@
 
 function calculateRentalCost(days) {
   // write code here
-}
+  const dayPrice = 40;
+  const sum = days * dayPrice;
+  const countDay0 = 0;
+  const countDay3 = 3;
+  const countDay7 = 7;
+  const shortRentDiscount = 20;
+  const longRentDiscount = 50;
+
+  if (days > countDay0 && days < countDay3) {
+    return sum;
+  } else if (days >= countDay3 && days < countDay7) {
+    return sum - shortRentDiscount;
+  } else if (days >= countDay7) {
+    return sum - longRentDiscount;
+  }
```

Comment:
Avoid `if / else` here. After return below code is unreachable, so it can be simplify. Also for future to improve your knowledge about [if-else hell](https://www.linkedin.com/pulse/avoid-if-else-hell-javascript-afsal-m-h/)
```suggestion
  if (days > countDay0 && days < countDay3) {
    return sum;
  } 
  
  if (days >= countDay3 && days < countDay7) {
    return sum - shortRentDiscount;
  }
  
 return sum - longRentDiscount;
```